plugins/scaffolder/src/components/MultistepJsonForm/MultistepJsonForm.tsx

```diff
@@ -49,6 +49,61 @@ type Props = {
   fields?: FormProps<any>['fields'];
 };
 
+export function getUiSchemasFromSteps(steps: Step[]): UiSchema[] {
+  const uiSchemas: Array<UiSchema> = [];
+  steps.forEach(step => {
+    if (!step.schema || !step.schema.properties) return [];
+
+    const schemaProps = step.schema.properties as JsonObject;
+    for (const key in schemaProps) {
+      if (Object.prototype.hasOwnProperty.call(schemaProps, key)) {
+        const uiSchema = schemaProps[key] as UiSchema;
+        uiSchema.name = key;
+        uiSchemas.push(uiSchema);
+      }
+    }
+  });
+  return uiSchemas;
+}
+
+export function getReviewData(formData: Record<string, any>, steps: Step[]) {
+  const uiSchemas = getUiSchemasFromSteps(steps);
+  const reviewData: Record<string, any> = {};
+  for (const key in formData) {
+    if (Object.prototype.hasOwnProperty.call(formData, key)) {
```

Comment:
also here? `formData.hasOwnProperty`? or does it not exist on the type?

Reply:
Another "mistake by autocompelte" :D